test/nstemplatetier/nstemplatetier.go

```diff
@@ -98,6 +98,26 @@ var CurrentBasicTemplates = toolchainv1alpha1.NSTemplateTierSpec{
 	},
 }
 
+// BaseTier returns a "base" NSTemplateTier with template refs in the given spec
+func BaseTier(t *testing.T, spec toolchainv1alpha1.NSTemplateTierSpec, options ...TierOption) *toolchainv1alpha1.NSTemplateTier {
```

Comment:
Is there any difference between this `BaseTier` and the `BasicTier` from below? If not, then maybe we can use `BasicTier` instead? If your intent was to update our code to use `BaseTier` then I guess we can rename the func in a separate PR?

Reply:
we don't have such `basic` tier so I thought that `base` would be more appropriate. But ok, let me use `BasicTier ` for now and I'll change to `BaseTier` in a separate PR

Reply:
done in 85579a0